rotors_description/urdf/firefly_base_gps.xacro

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
+<!--
+  Copyright 2015 Fadri Furrer, ASL, ETH Zurich, Switzerland
+  Copyright 2015 Michael Burri, ASL, ETH Zurich, Switzerland
+  Copyright 2015 Mina Kamel, ASL, ETH Zurich, Switzerland
+  Copyright 2015 Janosch Nikolic, ASL, ETH Zurich, Switzerland
+  Copyright 2015 Markus Achtelik, ASL, ETH Zurich, Switzerland
+  Copyright 2015 Marija Popovic, ASL, ETH Zurich, Switzerland
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<robot name="firefly" xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:include filename="$(find rotors_description)/urdf/component_snippets.xacro" />
+  <!-- Instantiate firefly "mechanics" -->
+  <xacro:include filename="$(find rotors_description)/urdf/firefly.xacro" />
+
+  <!-- Specify reference parameters for GNSS receiver. -->
+  <xacro:property name="referenceLatitude" value="47.3761890018" />
+  <xacro:property name="referenceLongitude" value="8.546860995" />
+  <xacro:property name="referenceHeading" value="0" />
+  <xacro:property name="referenceAltitude" value="0" />  
+
+  <!-- Instantiate a controller. -->
+  <xacro:controller_plugin_macro namespace="${namespace}" imu_sub_topic="imu" />
+
+  <!-- Mount an ADIS16448 IMU. -->
+  <xacro:default_imu namespace="${namespace}" parent_link="${namespace}/base_link" />
+
+  <!-- Mount a GNSS receiver. -->
+  <gazebo>
+    <plugin name="${namespace}_gps_sim" filename="libhector_gazebo_ros_gps.so">
```

Comment:
@ffurrer Okay, makes sense. Let me know if you guys think I should send a pull request there, instead!